pkg/common/labels.go

```diff
@@ -24,3 +24,10 @@ func GetLabels(name string, appLabel string) map[string]string {
 		"app":   appLabel,
 	}
 }
+
+// InitLabelMap - Inititialise a label map to an empty map if it is nil.
+func InitLabelMap(m *map[string]string) {
+	if *m == nil {
+		*m = make(map[string]string)
+	}
+}
```

Comment:
Since it can be used more generic I creatd an InitMap [1], so we won't need this one. Also seems not to be used.

[1] https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/pkg/common/map.go#L20